### PR TITLE
Kernel 3.5.0 Release Notes + Registry Functions

### PR DESCRIPTION
**Pull request by WillXuCodes**
Added the release blog for Kernel 3.5.0. Will merge when all other docs are ready for each of the features.

### CODE REVIEW

**Comment by WillXuCodes** [comment]
PR now contains new registry functions and the change regarding the IMU func being blocking.

File: v5/api/c/imu.rst [hunk 1]
```diff
@@ -15,7 +15,7 @@ imu_reset
 
 Calibrate IMU.
 
-This takes approximately 2 seconds, and is a non-blocking operation.
+This takes approximately 2 seconds, and is blocking until the IMU status flag is set properly.
```
Comment (nathan-moore) [inline v5/api/c/imu.rst:18]: I'd just say until the Imu is initialized. I'm not sure what the status flag is.

File: v5/extended/apix.rst [hunk 2]
```diff
@@ -33,12 +33,12 @@ Control settings of the way the file's driver treats the file.
...
 ------------------
 
-Registers a device of the given type in the given port into the registry, if
+Registers a device of the given type in the given zero-indexed port into the registry, if
 that type of device is detected to be plugged in to that port.
 
 This function uses the following values of ``errno`` when an error state is reached:
 
-- ``EINVAL``     - The given value is not within the range of V5 ports (1-21).
+- ``EINVAL``     - The given value is not within the range of V5 ports (0-20).
```
Comment (nathan-moore) [inline v5/extended/apix.rst:41]: Isn't every other port 1 indexed? (from a Glace that is what it looks like.) why is this different?

File: v5/api/cpp/imu.rst [hunk 3]
```diff
@@ -443,647 +443,6 @@ This function uses the following values of ``errno`` when an error state is reac
 
 ----
 
-tare_rotation
-~~~~~~~~~
```
Comment (nathan-moore) [inline v5/api/cpp/imu.rst:447]: Is this accidental?

File: v5/extended/apix.rst (hunk 2 above)
Comment (WillXuCodes) [reply v5/extended/apix.rst:41]: This change was made a to keep the iternal API consistent... turns out some of the functions were also documented incorrectly and it was decided that we keep them 0 indexed.

File: v5/api/c/imu.rst (hunk 1 above)
Comment (WillXuCodes) [reply v5/api/c/imu.rst:18]: file:///C:/Users/DELL-E~1/AppData/Local/Temp/Rar$EXa16940.23492/v5/api/c/imu.html?highlight=imu#imu-get-status , it's related to this status flag (I'll change it to state to be slightly more accurate though).

Comment (WillXuCodes) [reply v5/api/c/imu.rst:18]: It's related to imu_get_status, I guess I could change it to "until IMU status is set properly".

File: v5/api/cpp/imu.rst (hunk 3 above)
Comment (WillXuCodes) [reply v5/api/cpp/imu.rst:447]: yes, good catch lol.

**Pull request merged**